pages.de/common/npm.md

```diff
@@ -1,6 +1,6 @@
 # npm
 
-> Ein Kommandozeilenwerkzeug für die Verwaltung von Javascript und Node.js Paketen (Packages).
+> Ein Kommandozeilenwerkzeug für die Verwaltung von JavaScript und Node.js Paketen (Packages).
```

Comment:
Weird. I'd personally always write it `Javascript`, but apparently https://js.org/ agrees with you in this change :woman_shrugging:

Reply:
Yeah.  `Javascript` is definitely incorrect.  `JavaScript` is the official standard spelling.

Reply:
if it was Js when shortened it would make sense for it to be Javascript.
JS is definitely correct